packages/flutter/lib/src/material/radio_theme.dart

```diff
@@ -8,7 +8,9 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 
+import 'constants.dart';
 import 'material_state.dart';
+import 'radio.dart';
```

Comment:
@Hixie how does that work then? In the IDE, code navigation and highlighting in docs only works with the correct imports. In fact, it will add the imports automatically when writing the docs (and using IntelliSense).

Is it different for dartdoc? And are there downsides? And how does dartdoc know what to link to without the imports?